webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                raise ValueError(
+                    'Incorrect arguments. Either provide "csv files" or '
+                    '"ensembles and response_file".'
+                )
+            self.parameterdf = pd.read_parquet(self.parameter_csv)
+            self.responsedf = pd.read_parquet(self.response_csv)
+
+        elif ensembles:
+            self.ens_paths = {
+                ens: app.webviz_settings["shared_settings"]["scratch_ensembles"][ens]
+                for ens in ensembles
+            }
+            self.parameterdf = load_parameters(
+                ensemble_paths=self.ens_paths, ensemble_set_name="EnsembleSet"
+            )
+            if self.response_file:
+                self.responsedf = load_csv(
+                    ensemble_paths=self.ens_paths,
+                    csv_file=response_file,
+                    ensemble_set_name="EnsembleSet",
+                )
+            else:
+                self.responsedf = load_smry(
+                    ensemble_paths=self.ens_paths,
+                    column_keys=self.column_keys,
+                    time_index=self.time_index,
+                )
+                self.response_filters["DATE"] = "single"
+        else:
+            raise ValueError(
+                'Incorrect arguments.\
+                 Either provide "csv files" or "ensembles and response_file".'
+            )
+        self.check_runs()
+        self.check_response_filters()
+        if response_ignore:
+            self.responsedf.drop(response_ignore, errors="ignore", axis=1, inplace=True)
+        if response_include:
+            self.responsedf.drop(
+                self.responsedf.columns.difference(
+                    ["REAL", "ENSEMBLE", *response_include, *list(response_filters.keys()),]
+                ),
+                errors="ignore",
+                axis=1,
+                inplace=True,
+            )
+        if parameter_ignore:
+            self.parameterdf.drop(parameter_ignore, axis=1, inplace=True)
+
+        self.theme = app.webviz_settings["theme"]
+        self.badge_style = {
+            "font-weight": "bold",
+            "fontSize": ".70em",
+            "borderRadius": "3px",
+            "backgroundColor": "#505050",
+            "color": "white",
+            "white-space": "pre-wrap",
+        }
+        self.tooltip_style = {
+            "fontSize": ".75em",
+            "backgroundColor": "#505050",
+            "color": "white",
+            "opacity": "85%",
+            "white-space": "pre-wrap",
+            "borderRadius": "5px",
+        }
+        self.set_callbacks(app)
+
+
+    @property
+    def tour_steps(self):
+        """ Adding a "Guided tour" functionality """
+        steps = [
+            {
+                "id": self.uuid("layout"),
+                "content": (
+                    "Dashboard displaying the results of a multiple "
+                    "regression of input parameters and a chosen response."
```

Comment:
Similar to the documentation, it could be that a more comprehensive description of the algorithm that generates the results is useful. Keep in mind that most users will not read the code, but should still be able to understand possibilities, limitations and risks.